src/AzureStack/ComputeAdmin/Compute.Admin/Microsoft.AzureStack.Management.Compute.Admin.csproj

```diff
@@ -0,0 +1,15 @@
+﻿
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
```

Comment:
Please copy lines 2 and 4 from [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L2) (the comments are necessary)
Please do the same for FabricAdmin, CommerceAdmin and  InfrastructureInsightsAdmin

Reply:
Done.